### DISCUSSION TITLE
How to ensure secure service isolation

### DISCUSSION THREAD

**Discussion by zyiqian** [Kong/kong#13629] [Help]
hi team,
I am currently facing such a problem. When I request the domain name service1.com, and then modify the header information host: service2.com, the result is that I can enter the service2 service?

My KONG setup kong:3.1
here is my services configuration：
```
services: [
  {
    name: 'service1',
    host: 'service1',
    port: 80,
    routes: [
      {
        name: 'service1-route',
        protocols: ['https', 'http'],
        paths: [/v1]
        strip_path: false,
        hosts: [service1.com],
      },
    ],
  },
  {
    name: 'service2',
    host: 'service2',
    port: 80,
    routes: [
      {
        name: 'service2-route',
        paths: [/api]
        protocols: ['https', 'http'],
        strip_path: false,
        hosts: [service2.com],
      },
    ],
  },
]
```
I don't want them to be able to access each other by modifying the request headers. I want to ensure the secure isolation of the interfaces of the two services. What should I do?
can anyone help me here.

**Comment by chobits** [comment]
hi

> the result is that I can enter the service2 service?

This message lacks sufficient clarity for us to fully comprehend your issue, could you provide detailed steps to reproduce your problem, including specific commands. Additionally, please outline what you currently observe happening and what you expect to occur instead.
